scripts/irods/test/test_misc.py

```diff
@@ -337,6 +337,10 @@ def test_scoped_client_identity_updates_zone_of_RsComm__issue_6268(self):
     def test_scoped_permission__issue_7032(self):
         self.admin.assert_icommand(['irule', '-r', 'irods_rule_engine_plugin-irods_rule_language-instance', 'msi_test_scoped_permission', 'null', 'ruleExecOut'])
 
+    @unittest.skipUnless(plugin_name == 'irods_rule_engine_plugin-irods_rule_language', "Not implemented for other REPs.")
+    def test_user_administration__issue_NNNN(self):
```

Comment:
Issue number?

Reply:
Nice catch.

So easy to forget things.

Reply:
Done